src/Microsoft.ML.Sweeper/Microsoft.ML.Sweeper.csproj

```diff
@@ -6,6 +6,10 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="System.Threading.Channels" Version="$(SystemThreadingChannelsPackageVersion)" />
```

Comment:
This shouldn't be needed because this project references `Microsoft.ML.Data`, which references this package.